src/xmlTags.ts

```diff
@@ -30,18 +44,34 @@ import {
   TSpan,
   Use,
 } from './elements';
+import FeDistantLight from './elements/filters/FeDistantLight';
```

Comment:
Why only this is imported in a different style?

Reply:
Fixed in [b3fbd3c](https://github.com/software-mansion/react-native-svg/pull/2392/commits/b3fbd3c56649435baf1b59531623a17275ddd12d)